src/apps/command_index_app.tsx

```diff
@@ -10,22 +10,24 @@ import SystemsService from "../services/system_service";
 import Table from "../components/table";
 import System from "../custom_types/system_type";
 import Command from "../custom_types/command_type";
+import CacheService from "../services/cache_service";
+import TableService from "../services/table_service";
+import MyState from "../custom_types/table_state_type";
 
 type MyProps = {
   systems: System[];
   match: any;
 };
-type MyState = {
-  data: any[];
-  page: number;
+
+type CachedStateType = {
```

Comment:
This definition shows up in multiple places. Are you moving this to some central place as part of your type cleanup branch?

Reply:
I took a look in the what I am doing for issue #34 and will be changing its location in a separate branch.